### ISSUE TITLE
Screen reader is announcing incorrect name for the 'Arrow' control present in the table in Windows.

### ISSUE THREAD

**Issue by msftedad** [radzenhq/radzen-blazor#2178]
**Describe the bug**
On pressing ctrl + alt + arrow keys, when focus moves to the arrow control screen reader announces it as 'Column 1 arrow dropdown' in Windows.
Screen reader users will not get the proper context of the control to interactive control if it announces incorrect name.

**To Reproduce**

1. Open the URL: [Blazor ProgressBar Component | Free UI Components by Radzen](https://blazor.radzen.com/progressbar?theme=material#progressbar-determinate) in latest edge browser.
2. Progress bar page will appear.
3. Press tab key to reach to the progress bar.
4. Verify on pressing arrow key, whether screen reader is announcing incorrect name for the 'Arrow' control present in the table or not.

Alternatively link your repo with a sample project that can be run.
https://blazor.radzen.com/progressbar?theme=material#progressbar-determinate

**Expected behavior**
On pressing ctrl + alt + arrow keys, when focus moves to the arrow control screen reader should announce it as 'Column 1 expand child item' in Windows.

**Screenshots**

https://github.com/user-attachments/assets/4585e81c-6ee0-4344-830b-0854b647d03f

<img width="976" alt="Image" src="https://github.com/user-attachments/assets/e2f0af3d-81eb-4638-9dd1-5378d2c61662" />

**Desktop (please complete the following information):**
OS: Windows 11 Version 24H2(OS Build 26120.1930)
Browser: Edge Version 130.0.2849.27

**Additional context**
NA

**Comment by enchev** [comment]
Not sure how the progress bar is related to this?
